Automattic/wp-calypso
# Concept: Move theme setup operation to the end of flow

## Proposed Changes

* Community themes such as Course are not allowed to be installed on sites without Creator or Higher plans. So if we try to install them earlier, it can be problematic and can lead to failure if (or when) this check is strictly imposed in the backend. So here we're moving it to the end of our flow.

#### Known issue:

The style variation we get from redux store ( `getSelectedStyleVariation` ) is empty at this stage because redux store gets cleared. We need a way to persist the variation selection.

## Testing Instructions



* Go to `http://calypso.localhost:3000/setup/sensei` and complete the onboarding flow.
* Make sure it completes without any issue and Course theme is set as the theme of the site.
* Check for both new and existing user
* Check with and without proxy

## Pre-merge Checklist
